tests/integration/helpers/integration-test-helper.js

```diff
@@ -136,6 +136,16 @@ TestUtility.prototype.VerifyFileContents = function (dir, fileName, expectedCont
     var _this = this;
     _this.runFunc(function () {
         var bundleContents = _this.FileSystem.readFileSync(dir + '/' + fileName, 'utf8');
+        if (expectedContents.length !== bundleContents.length) {
```

Comment:
@Andrew-Hagedorn-ZocDoc Removing the whitespace before comparing ended up being a little weird. Instead, I've added some console logging that helped me pinpoint exactly what the mismatch in the string was. Turns out it was the newline characters.